dora/tests/integration/src/test/java/alluxio/client/hadoop/DFSIOIntegrationTest.java

```diff
@@ -660,7 +660,8 @@ public Closeable getIOStream(String name) throws IOException {
     public Long doIO(Reporter reporter, String name, long totalSize) throws IOException {
       PositionedReadable in = (PositionedReadable) this.mStream;
       long actualSize = 0;
-      for (long pos = nextOffset(-1); actualSize < totalSize; pos = nextOffset(pos)) {
+      for (long pos = nextOffset(-1); actualSize < totalSize
+          && pos < totalSize; pos = nextOffset(pos)) {
```

Comment:
Which ignored test do you fix? Frankly I don't understand the necessity of this modification. Existing tests are passed whether modify the two lines or not.